test/ServerComparison.FunctionalTests/ResponseCompressionTests.cs

```diff
@@ -96,13 +98,34 @@ public Task ResponseCompression_AppAndHostCompression(TestVariant variant)
                     ApplicationPath = Helpers.GetApplicationPath(),
                     EnvironmentName = "ResponseCompression",
                 };
-                if (hostCompression)
+
+                if (variant.Server == ServerType.Nginx)
                 {
-                    deploymentParameters.ServerConfigTemplateContent = Helpers.GetNginxConfigContent(variant.Server, "nginx.conf");
+                    deploymentParameters.ServerConfigTemplateContent = hostCompression
+                        ? Helpers.GetNginxConfigContent("nginx.conf")
+                        : Helpers.GetNginxConfigContent("NoCompression.conf");
                 }
-                else
+                else if (variant.Server == ServerType.IISExpress)
                 {
-                    deploymentParameters.ServerConfigTemplateContent = Helpers.GetConfigContent(variant.Server, "NoCompression.config", "NoCompression.conf");
+                    var iisDeploymentParameters = new IISDeploymentParameters(deploymentParameters);
+                    iisDeploymentParameters.ServerConfigActionList.Add(
+                        (element, _) => {
+                            var compressionElement = element
+                                .RequiredElement("system.webServer")
+                                .RequiredElement("httpCompression");
+
+                            compressionElement
+                                .RequiredElement("dynamicTypes")
+                                .Elements()
+                                .SkipLast(1)
```

Comment:
??

Reply:
This is exactly the change NoCompression.config did - removing all but last element from `dynamicTypes` and `staticTypes`

Reply:
![image](https://user-images.githubusercontent.com/1697911/44228229-a05e0180-a149-11e8-9885-676a0f7b7b29.png)

Reply:
Deserves a comment about how that last element disables compression

Reply:
Done